[rospy.client][INFO] 2020-11-12 06:10:26,546: init_node, name[/train_9306_1605161426539], pid[9306]
[xmlrpc][INFO] 2020-11-12 06:10:26,547: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-12 06:10:26,548: Started XML-RPC server [http://luigi:44657/]
[rospy.init][INFO] 2020-11-12 06:10:26,548: ROS Slave URI: [http://luigi:44657/]
[rospy.impl.masterslave][INFO] 2020-11-12 06:10:26,548: _ready: http://luigi:44657/
[rospy.registration][INFO] 2020-11-12 06:10:26,550: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-12 06:10:26,550: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-12 06:10:26,649: registered with master
[rospy.rosout][INFO] 2020-11-12 06:10:26,649: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-12 06:10:26,655: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-12 06:10:26,656: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-12 06:10:26,676: topic[/bev/flow_image] adding connection to [http://192.168.0.201:44563/], count 0
[rospy.internal][INFO] 2020-11-12 06:10:26,681: topic[/odom] adding connection to [http://192.168.0.201:46639/], count 0
[rospy.internal][INFO] 2020-11-12 06:10:26,682: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:44563/], count 0
[rospy.internal][INFO] 2020-11-12 06:10:26,941: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-11-12 06:10:56,428: signal_shutdown [signal-2]
[rospy.internal][INFO] 2020-11-12 06:10:56,432: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-12 06:10:56,433: topic[/bev/flow_image] removing connection to http://192.168.0.201:44563/
[rospy.internal][INFO] 2020-11-12 06:10:56,433: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:44563/
[rospy.internal][INFO] 2020-11-12 06:10:56,434: topic[/odom] removing connection to http://192.168.0.201:46639/
[rospy.impl.masterslave][INFO] 2020-11-12 06:10:56,434: signal-2
[rospy.internal][WARNING] 2020-11-12 06:10:56,436: Unknown error initiating TCP/IP socket to 192.168.0.201:35321 (http://192.168.0.201:46639/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.core][INFO] 2020-11-12 06:10:56,928: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 06:10:56,970: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 06:10:56,991: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 06:10:57,022: signal_shutdown [signal-2]
[rospy.core][INFO] 2020-11-12 06:10:57,030: signal_shutdown [atexit]
